### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sreeramg514**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # IntegrityasaService
-Creating integrity between retailer and customer
+Adding integrity between retailer and customer
```
Comment (sreeram514) [inline README.md:2]: dddd

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # IntegrityasaService
-Creating integrity between retailer and customer
```
Comment (sreeram514) [inline README.md:2]: dsaa

File: README.md (hunk 1 above)
Comment (sreeramg514) [reply README.md:2]: ddfdd

**Review by sreeram514 — COMMENTED** [review]
ddddd3e2

**Pull request merged**